backend/tradingbot/tests.py

```diff
@@ -1,11 +1,58 @@
 from django.urls import reverse
-from rest_framework.test import APITestCase
 from rest_framework import status
+from rest_framework.test import APITestCase
+
+from .models import Company, StockTrade
 
 
 class TradingbotTests(APITestCase):
 
+    def setUp(self) -> None:
+        company = Company(
+            name="Apple",
+            ticker="AAPL",
+        )
+        company.save()
```

Comment:
Can't do `Company(.....).save()` because `.save()` returns None and `Company` object is needed to create new `StockTrade` instance

Reply:
what does .save() do (and why do we need it)? I am a little bit confused about this part

Reply:
When you create a model instance with `model = Model(**kwargs)` it's only created in local memory, the corresponding row isn't created in the database until `model.save()`.
When one model "A" has a foreign key to another model "B" you need to pass an instance of "B" into "A"